packages/angular_devkit/build_angular/src/differential-loading/utils.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import * as browserslist from 'browserslist';
+import * as caniuse from 'caniuse-api';
+import * as path from 'path';
+import { ScriptTarget } from 'typescript';
+import { readTsconfig } from '../angular-cli-files/utilities/read-tsconfig';
```

Comment:
This will most likely be removed in an upcoming refactoring/cleanup.  You can use `ts.getParsedCommandLineOfConfigFile` directly.